addons/project/views/project_views.xml

```diff
@@ -20,7 +20,7 @@
                <search string="Tasks">
                     <field name="name" string="Tasks" filter_domain="['|', ('name', 'ilike', self), ('id', 'ilike', self)]"/>
                     <field name="tag_ids"/>
-                    <field name="user_ids" filter_domain="[('user_ids', 'ilike', self), ('user_ids.active', 'in', [True, False])]"/>
+                    <field name="user_ids" filter_domain="[('active', '=', True), ('user_ids', 'ilike', self), ('user_ids.active', 'in', [True, False])]" context="{'active_test': False}"/>
```

Comment:
But then if we search a user and then apply `Archived` filter we will have no records. 😅

Reply:
Yup, you are right, I will discuss with the framework-PY, else it's a limitation.

Reply:
Well good news @xavierbol I've found a solution, I've pushed the fix.
I've updated the PR and commit message for the reason. :)